scio-macros/src/main/scala-3/com/spotify/scio/IsJava.scala

```diff
@@ -28,7 +28,7 @@ object IsJavaBean {
 
   private def checkGetterAndSetters(using q: Quotes)(sym: q.reflect.Symbol): Unit = {
     import q.reflect._
-    val methods: List[Symbol] = sym.classMethods
+    val methods: List[Symbol] = sym.declaredMethods
```

Comment:
Is it possible that the methods are inherited, thus not in `declaredMethods`?

Reply:
Actually, yes, they could be. We should be using `memberMethods`, not `declaredMethods`

Reply:
Opened #3 to track this.